integration-tests/src/test/projects/maven-conf-ignore-ext-def/pom.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+
+    Copyright 2019-2022 the original author or authors.
```

Comment:
Remove the copyright ?

Reply:
every IT POM has it.... so maybe a sweeping change to remove them all and leave plain ASL2 header?